doc/skrub_pipeline.rst

```diff
@@ -0,0 +1,353 @@
+.. _skrub_pipeline:
+
+===================================================
+Skrub Pipeline: flexible machine learning pipelines
```

Comment:
I hate to be that person, but that title is too generic: it does not make us guess what is different to other pipelines.

The cool thing about the skrub pipeline is that it applies to any wrangling tool.

Maybe "Cross-validate or tune any wranging pipeline"

Reply:
Agreed, was looking for something better. Let's use your suggestion

Reply:
What about "Cross-validate or tune any **data** pipeline", since I'm not sure most readers will be familiar with wrangling pipelines